webpack.config.js

```diff
@@ -2,6 +2,11 @@ const path = require('path');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 
 module.exports = {
+  watch: true,
```

Comment:
- Give the settings to your webpack. You can have this for [reference](https://createapp.dev/).